### ISSUE TITLE
Wifi does not reset after closing/reopening lid [WORKAROUND]

### ISSUE THREAD

**Issue by matt32106** [Grippentech/Asus-E200HA-Linux-Post-Install-Script#21]
If you close the lid the wifi shuts down. 
When you repoen the lid:
- it does not restart automatically 
- if i use the icon at the top right of the gnome interface, it does not work neither

Workaround: open a terminal (ctrl-alt-T) and 
`sudo service network-manager restart`

Ubuntu 16.10
uname -r > 4.12.0-rc1-custom3

**Comment by federeghe** [comment]
For me it works correctly, maybe a kernel version regression?

Arch Linux
uname -r > 4.10.0-arch-cstm+

**Comment by Grippy98** [comment]
Yeah this is a known issue with it right now. It seems to only really happen in Ubuntu (that I've experienced). 

And yeah, restarting the network manager service does the trick. I'm not sure what causes this...

**Comment by matt32106** [comment]
is it gnome specific?

**Comment by Grippy98** [comment]
No, it seems to happen under Unity too. I haven't had it happen in LXDE that I can remember though. 

I really don't understand why some bugs crop up under some desktop environments but not others.

Also if you check the readme I gave u credit for the networking restart temp-fix since a bunch of users were complaining about that happening and having to restart :)

**Comment by matt32106** [comment]
oh thanks but i discovered after posting that the fix was already in one of the issues... :)
it's damienfern that should be credited

can it be that each desktop uses different applets?!

**Comment by Grippy98** [comment]
I'm not sure to be honest... it's weird since the same version of something like PulseAudio will outright crash on one environment and work perfectly in another.
